community/howto/120decibell/calculating_driver_current.md

```diff
@@ -6,22 +6,25 @@ nav_exclude: true
 
 # Calculating Driver Current Settings
 
-When using the 2208 / 2209 drivers, the voltage & current are set in software.  In Klipper, the motor currents have two settings: run and hold.  [However, it is no longer recommended][klipperTMCUpdate] to specify a `hold_current` for most motors. Klipper current settings are based on root-mean-squared (RMS) and **not** on peak current.  The spec sheets for most motors list the peak current capacity.
+When using the 2208 / 2209 drivers, the voltage & current are set in software.  In Klipper, the motor currents have two settings: run and hold.  [However, it is no longer recommended][klipperTMCUpdate] to specify a `hold_current` for most motors. Klipper current settings are based on root-mean-squared (RMS) and **not** on peak current.  Depending on the spec sheet the rated current is either peak or RMS.  If you are unsure, assume that it is peak.
 
 [klipperTMCUpdate]: https://www.klipper3d.org/TMC_Drivers.html#prefer-to-not-specify-a-hold_current
 
 #### Calculating Currents
 
-To calculate the maximum Klipper current settings for a given stepper, follow this process:
+To calculate the maximum Klipper current settings for a given stepper assuming peak specifications, follow this process:
 
 1. Look up the specifications for the stepper motor and locate the peak current limits of the motor.
 2. Multiply the peak current by 0.707 to determine the maximum current in RMS.  This is the maximum run current. (Typically round down to the nearest .1)
 3. If you plan to set a separate hold current, multiply the maximum run current by 0.6 to determine the hold current.  (Typically round to the nearest (0.05)
+4. Both value needs be less than 1.2 amps which recommended maximum value for TMC 2209 drivers.
```

Comment:
It is recommended to keep the current below 1.2 amps which is the recommended maximum value for stepstick form factor TMC 2209 drivers

1. remove both values since we are getting rid of hold current
2. mention the limit is for stepstick form factor 2209s (I've heard 1.2A too, but is there a reference for that? We could back it up somehow with a link)

Reply:
Will do.